manifests/init.pp

```diff
@@ -714,7 +714,7 @@
   Optional[Stdlib::Absolutepath] $server_puppet_basedir = $puppet::params::server_puppet_basedir,
   Enum['current', 'future'] $server_parser = $puppet::params::server_parser,
   Variant[Undef, Enum['unlimited'], Pattern[/^\d+[smhdy]?$/]] $server_environment_timeout = $puppet::params::server_environment_timeout,
-  String $server_jvm_java_bin = $puppet::params::server_jvm_java_bin,
+  Optional[Stdlib::Absolutepath] $server_jvm_java_bin = undef,
```

Comment:
This will need an installer migration as well to wipe the value. I wonder if this a breaking change, but I think a relative path was never valid.

Reply:
Agreed.

Reply:
https://github.com/theforeman/foreman-installer/pull/922